gui-apps/xwaylandvideobridge/xwaylandvideobridge-9999.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+KFMIN=5.83.0
+QTMIN=5.15.0
```

Comment:
kpipewire depends on `QTMIN=5.15.7` and `KFMIN=5.102.0` respectively so it makes no sense to go below those here.